lib/iex/resources/resource.rb

```diff
@@ -17,6 +17,41 @@ def self.float_to_percentage(float_number)
       def self.to_dollar(amount:, ignore_cents: true)
         MoneyHelper.money_to_text(amount, 'USD', nil, no_cents: ignore_cents)
       end
+
+      def self.to_full_issue_type(type)
```

Comment:
As for me, it's more specific to the `Symbols` class. When you need to do simple mapping it's better to use `Hash` for it:
```ruby
class Symbols < Resource
  TYPE_DESCRIPTIONS = {
    'ad' => 'ADR',
    're' => 'REIT'
     ...
  }
...
  property 'type_description', from: 'type', with: ->(type) { TYPE_DESCRIPTIONS[type] }
...
```
Such long strings sound more like type description. When I wrote the comment about `full_type` I meant a field that you can use in your app for example for filtering. Let's imagine you need only common stocks. As for me, more readable to have code like this:
```
symbols.select { |symbol| symbol[:full_type] == :common_stock }
```
instead of:
```
symbols.select { |symbol| symbol['full_type'] == 'cs' }
```

It's just a suggestion. You can do it if you feel that it will be helpful.

Reply:
This type itself is not a symbol-specific value.

For example, it's also used by SEARCH.

https://iexcloud.io/docs/api/#search

>refers to the common issue type (AD - ADRRE - REITCE - Closed end fundSI - Secondary IssueLP - Limited PartnershipsCS - Common StockET - ETFWT - WarrantOEF - Open Ended FundCEF - Closed Ended FundPS - Preferred Stock)UT- UnitTEMP - Temporary


So I think it is correct that it is in Resource.

It's nice to do it the HASH way 😁

After all, I don't need this method itself, so I don't want to put it in this time.

I would like you to add it if you need it.